@pranathi-golla , you've got one extra blank line that is keeping this from passing CI! :)

I would just fix it myself, except I don't think I have access to the branch in your fork.

```
Run flake8 python --count --max-complexity=25 --max-line-length=127 --statistics
python/parser.py:831:1: W391 blank line at end of file
1     W391 blank line at end of file
1
Error: Process completed with exit code 1.
```

--- a/python/parser.py
+++ b/python/parser.py
@@ -828,4 +828,3 @@ if __name__ == "__main__":
     example_control.generate_plant_uml(
         filename="testing/ExampleControl-Dead.txt", top="Dead"
     )
-

